swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+#from swap_meet.vendor import Vendor
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self,category="",condition=0,age=0):
+        self.category=category
+        self.condition=condition
+        self.age=age
+
+    def __str__(self):
+        return (f"Hello World!")   
+
+    def condition_description(self): 
+        cond_description=self.condition
+        if cond_description==0:
+            return ("very bad")
+        elif cond_description==1:
+            return("moderate")
+        elif cond_description==2:
+            return("normal")
+        elif cond_description==3:
+            return("acceptable")    
+        elif cond_description==4:
+            return ("good")
+
+        return ("very very good")    
```

Comment:
The condition can be a float, so it can contain a decimal.  In the cases where the condition has a decimal, this function will return "very very good".  There are a few solutions, one is to cast the condition to an int, which will drop the decimal portion of the number and round down.   Another possible solution is to test for a range of values (ex: `if self.condition <= 5 and self.condition > 4`.